docs/application/dotnet/guides/nui/styling-controls-with-xaml.md

```diff
@@ -0,0 +1,201 @@
+  </ViewStyle.BackgroundColor>
+</ViewStyle>
+```
+
+### Load theme in NUI application
+
+To use the theme implemented in XAML, follow these steps: 
+
+1. Create a theme object as follows:
+     ```csharp
+    Theme theme = new Theme("PATH"); \\ where PATH is a full path to the XAML file in a project.
+     ```
+2. Use `ThemeManager` to apply the theme as follows:
+    ```csharp
+   ThemeManager.ApplyTheme(theme);
+    ```
+3. Use style defined in the XAML file in the NUI component by setting the `StyleName` parameter as follows:
+    ```csharp
+    component.StyleName = <xaml_style_name>;
+    ```
+
+The following example explains the implementation of the `TextLabel` NUI component:
+
+```csharp
+using Tizen.NUI; 
+
+namespace NUI_Theme
+{
+  class Program : NUIApplication
+  {
+    protected override void OnCreate()
+    {
+      // Load theme from the xaml file
+      Theme theme = new Theme(<PATH>); \\<PATH> is a full path to the XAML file in a project.
+
+      // Apply it to the current NUI Application
+      ThemeManager.ApplyTheme(theme);
+
+      // Set style name in NUI widget
+      TextLabel textLabel = new TextLabel()
+      textLabel.StyleName = "redTextStyle";
+    }
+  }
+```
+
+### Handle theme changes in runtime
+
+The following methods can be used to handle theme changes in runtime:
+
+- `ThemeManager.ThemeChanged`
+- `View.OnThemeChanged`
+
+When `ThemeManager.applyTheme` is called, `ThemeManager` calls the `ThemeChanged` callback. The following example shows how to handle `ThemeChanged` events:
+
+```csharp
+using Tizen.NUI;
+
+namespace NUI_Theme
+{
+  class Program : NUIApplication
+  {
+    protected override void OnCreate()
+    {
+      //application init
+
+      ThemeManager.ThemeChanged += OnThemeChanged;
+    }
+
+    private void OnThemeChanged(object sender, ThemeChangedEventArgs args) 
+    {
+      //Implement reaction on theme change here.
+    }
+  }
+}
+```
+
+The `View.OnThemeChanged` is used when you create a class that is derived from `View`. You can overload `OnThemeChanged` so that all functionalities related to the looks of the widget are implemented in one class:
+
+```csharp 
+class AlwaysBlueTextLabel : TextLabel
+{
+  protected override void OnThemeChanged(object sender, ThemeChangedEventArgs e)
+  {
+    // Theme has changed but the new style is not applied to this view yet.
+
+    // Call base method to apply new style in new theme to this view.
+    base.OnThemeChanged(sender, e);
+
+    // Set text color to blue.
+    textLabel.TextColor = Color.Blue;
+  }
+}
+```
+
+## Additional information:
+  - [XAML syntax](https://github.com/dalihub/nui-demo/blob/master/ThemeExample/docs/NUIXamlStyleSyntax.md)
+  - [XAML theme manager](https://github.com/dalihub/nui-demo/tree/master/ThemeExample/Basic1)
+  - [XAML Todo List](https://github.com/dalihub/nui-demo/tree/master/ThemeExample/TodoList)
+
+## Dependencies
+  -   Tizen 6.5 and Higher
```

Comment:
@mihashco, since this content is for 6.5, you must raise this PR for tizen_6.5_prepare branch. Request you to rebase and raise the PR against that branch.

Reply:
@annie-samsung OK. If you will merge this branch could you please squash commits?